pages/recipe/update/[id].js

```diff
@@ -1,21 +1,59 @@
 import { useRouter } from "next/router";
 import { useState } from "react";
-import CreateOrUpdateRecipeForm from "@/components/CreateOrUpdateRecipeForm";
 import { prepareFormData } from "@/utils/utils";
+import CreateOrUpdateRecipeForm from "@/components/CreateOrUpdateRecipeForm";
 
 export default function UpdateRecipeDetails({ recipes, onUpdateRecipe }) {
   const [error, setError] = useState("");
   const [inputValidation, setInputValidation] = useState("");
+  const [image, setImage] = useState();
+  const [toDelete, setToDelete] = useState(false);
   const router = useRouter();
   const { id } = router.query;
 
   const recipeDetails = recipes.find((recipe) => recipe.id === id);
 
-  function handleUpdateSubmit(event) {
+  if (recipeDetails?.imageURL && (image === null || undefined)) {
+    setImage({ id: recipeDetails.imageURL, toDelete: false });
+  }
```

Comment:
Are you wanting to check if `image` is equal to null or if `image` is equal to undefined?

If so i think you want to do this...

``` js
if (recipeDetails?.imageURL && (image === null || image === undefined)) {
  setImage({ id: recipeDetails.imageURL, toDelete: false });
}
```